docker-app/qfieldcloud/subscription/models.py

```diff
@@ -627,6 +649,38 @@ def active_users_count(self) -> int:
 
         return self.active_users.count()
 
+    @property
+    def organization_members_count(self) -> int:
+        # if non-organization account, then it is always 1 user
+        if not self.account.user.is_organization:
+            return 1
+
+        return OrganizationMember.objects.filter(
+            organization_id=self.account.user.pk
+        ).count()
+
+    @property
+    def included_storage_mb(self) -> float:
+        """
+        How much storage (in MB) this subscription comes with by default.
+        - For the 'flat' plan (1GB per seat) is calculated as seats x plan.storage_mb
+        - For any other plan, just plan.storage_mb
+        """
```

Comment:
```suggestion
        """
        How much storage (in MB) this subscription comes with by default.
        - For the 'flat' plan (1GB per seat) is calculated as `Subscription.max_organization_members * Plan.storage_mb`
        - For any other plan, just use `Plan.storage_mb`
        """
```

Same applies to `included_storage_bytes`.

Reply:
Actually second though, can we skip adding this method at all, I saw that `active_storage_total_mb` was already marked as deprecated. In general there is a push to kill the megabytes and use only bytes in calculations, re int/float conversation from the previous comment.